msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -141,6 +140,9 @@ As in Microsoft Teams, tab applications are hosted within [iframe elements](http
 
 If your app makes use of [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) (CSP) headers, make sure you allow all the following [frame-ancestors](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors) in your CSP headers:
 
+> [!WARNING]
```

Comment:
Instead of adding duplicate warnings to this article and `tab-requirements`, how about writing the warning once in an include file? Since both of these articles reference `content-security-policy-headers` include file already, I suggest incorporating the warning and CTAs into that include file

Reply:
Done